src/usr/local/www/system_advanced_network.php

```diff
@@ -186,6 +199,15 @@
 ))->setHelp('By default, if IPv6 is configured and a hostname resolves IPv6 and IPv4 addresses, '. 
 	'IPv6 will be used. If this option is selected, IPv4 will be preferred over IPv6.');
 
+$section->addInput(new Form_Input(
+	'global-v6duid',
+	'DHCP6 DUID',
+	'text',
+	$pconfig['global-v6duid'],
+	['placeholder' => 'xx:xx:xx:xx:xx:xx:xx:xx:xx:xx:xx:xx:xx:xx']
+	))->setWidth(9)->sethelp('Enter the DUID to use here. Use this option if using RAM Disk, as the DUID will be lost on reboot. The existing DUID may be found in var/db/dhcp6_duid.' .'<br />' .
+			'NOTE: Do not use this option with multiple DHCP6 WAN interfaces.');
```

Comment:
I will bow to your greater knowledge on that then.

Problem is, on RAM disks the DUID disappears on reboot, a new one is created, but that only happens when dhcp6 is fired up, I can see all sorts of 'I've got no DUID' type questions.  I'll have a look at pulling the duid from the file and inserting it in the placeholder. If nothing else I'm finding out more about pfSense..